Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -347,6 +346,42 @@ lemma gaussianReal_mul_const {X : Ω → ℝ} (hX : Measure.map X ℙ = gaussian
 
 end Transformations
 
+open Measurable Real
+
+variable {Ω : Type*} {mΩ : MeasurableSpace Ω} {p : Measure Ω} {μ : ℝ} {v : ℝ≥0} {X : Ω → ℝ}
+
+lemma mgf_dirac (hX : p.map X = .dirac μ) (t : ℝ) : mgf X p t = exp (μ * t) := by
```

Comment:
One last request, and then I finally merge this PR: this lemma is not about Gaussians. Could you move it to the Moments file?